.env.development

```diff
@@ -1,5 +1,5 @@
 ENV=dev
-
+PROBATE_FEES_RELEASE_DATE=2017-05-01
```

Comment:
this gets loaded only on development envs. We can se the PROBATE_FEES_RELEASE_DATE variable to any date we like during the staging deployment.